specification/cognitiveservices/data-plane/Face/readme.python.md

```diff
@@ -6,21 +6,23 @@ Use `--python-mode=update` if you already have a setup.py and just want to updat
 
 ``` yaml $(python)
 python-mode: create
-python:
-  license-header: MICROSOFT_MIT_NO_VERSION
-  add-credentials: true
-  payload-flattening-threshold: 2
-  namespace: azure.cognitiveservices.vision.face
-  package-name: azure-cognitiveservices-vision-face
-  clear-output-folder: true
+license-header: MICROSOFT_MIT_NO_VERSION
+add-credentials: true
+payload-flattening-threshold: 2
+namespace: azure.cognitiveservices.vision.face
+package-name: azure-cognitiveservices-vision-face
+clear-output-folder: false
+use: "@microsoft.azure/autorest.python@~4.0.71" 
+version: V2
+multiapi: true
+no-async: true
```

Comment:
Autorest will generate the completely wrong set of files without these settings.